Library/Formula/libgdiplus.rb

```diff
@@ -0,0 +1,32 @@
+class Libgdiplus < Formula
+  desc "C-based implementation of the GDI+ API for Mono"
+  homepage "http://www.mono-project.com/docs/gui/libgdiplus/"
+  url "https://github.com/mono/libgdiplus/tarball/3b284ab28cb8737f9d14dabfedc6903655c66a7f"
+  head "https://github.com/mono/libgdiplus.git"
+  sha256 "c7b6e68f4f4ef62e1f7551769c7f0b87e7debd52311123a0264d23cf7ac9aee8"
+  version "3.12" # latest tag = 3.12, plus two patches.
+
+  # it depends on mono to link to System.Drawing
+  depends_on "mono"
```

Comment:
I think instead of making this depend on mono, it should be installed as a recommended resource inside mono. Thus, it will have same behavior as upstream installer.